mephisto/abstractions/providers/mturk/mturk_datastore.py

```diff
@@ -122,6 +130,7 @@ def new_hit(self, hit_id: str, hit_link: str, duration: int, run_id: str) -> Non
                 ) VALUES (?, ?);""",
                 (hit_id, run_id),
             )
+            self._last_hit_mapping_update_time = time.time()
```

Comment:
nit: maybe define a new function for all the updates like this. In case you wanted to log updates or something later. For example,
```
def set_cache_updated():
  self._last_hit_mapping_update_time = time.time()
```